packages/module/src/components/VisualizationSurface.tsx

```diff
@@ -57,27 +57,31 @@ const VisualizationSurface: React.FunctionComponent<VisualizationSurfaceProps> =
   );
 
   // dispose of onMeasure
-  React.useEffect(() => () => clearTimeout(timerId.current), [onMeasure]);
+  useEffect(() => () => clearTimeout(timerId.current), [onMeasure]);
 
   if (!controller.hasGraph()) {
     return null;
   }
 
   const graph = controller.getGraph();
 
+  // TODO: We need to replace react-measure as it doesn't support React 19. The following
+  // casting helps to get topology to build with React 19 versions
+  const Wrapper = ReactMeasure as any;
```

Comment:
Just FYI, I opened a followup issue regarding this: https://github.com/patternfly/react-topology/issues/283